snap/snapcraft.yaml

```diff
@@ -122,21 +135,47 @@ parts:
     - qt6-base-dev-tools
     stage-packages:
     - apparmor
-    - on amd64: [libgl1]
-    - on armhf: [libgles2-mesa]
-    - on arm64: [libgles2-mesa]
-    - libc6
     - libpng16-16
     - libqt6core6
     - libqt6gui6
     - libqt6network6
     - libqt6widgets6
     - libssl3
     - libxml2
-    - libxrender1
     - dnsmasq-base
     - dnsmasq-utils
     - qt6-qpa-plugins
+    stage:
+    - -**/glvnd
+    - -**/X11
+    - -usr/bin/aa-*
+    - -usr/include
+    - -usr/lib/cmake
+    - -usr/lib/*/libEGL_mesa.so.*
+    - -usr/lib/*/libGLX_mesa.so.*
+    - -usr/lib/*/libGLX_indirect.so.*
+    - -usr/lib/*/libXxf86vm.so.*
+    - -usr/lib/*/libxcb-dri2.so.*
+    - -usr/lib/*/libxcb-present.so.*
+    - -usr/lib/*/libxshmfence.so.*
+    - -usr/lib/*/libicuio.so.*
+    - -usr/lib/*/libicutest.so.*
+    - -usr/lib/*/libicutu.so.*
+    - -usr/lib/*/libLLVM-15.so*
+    - -usr/lib/*/libdrm*
```

Comment:
How did you find the different parts of a package? And then know which parts we actually need?

Reply:
Again, it was the Snapcraft linter giving some clues along with trial and error.